sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -73,7 +74,14 @@ def get_enabled_regions() -> Set[str]:
     global regions_enabled
     if regions_enabled is None:
         aws_client = aws.client('ec2', region_name='us-east-1')
-        user_cloud_regions = aws_client.describe_regions()['Regions']
+        try:
+            user_cloud_regions = aws_client.describe_regions()['Regions']
+        except aws.botocore_exceptions().ClientError:
+            with ux_utils.print_exception_no_traceback():
+                raise RuntimeError(
+                    'Failed to retrieve AWS regions. '
+                    'Please ensure that the `ec2:DescribeRegions` action '
+                    'is enabled for your AWS account.') from None
```

Comment:
Is there a link to console page that can change this permission we can add here?

Also, maybe add: "Ref: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_DescribeRegions.html"

Reply:
I don't think it's possible to add a direct link to the console page because this will depend on the specific policy assigned to the user. However, I've updated the message to refer specifically to IAM so this is clearer. Added the link too!